Tools-Override/depProj.targets

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+***********************************************************************************************
+depProj.targets
```

Comment:
We added the `FilterNugetPackages` target so that we can use PackagesToInclude / PackagesToExclude in order to filter the restored files we get from NuGet.